janitor/cloudbuild.yaml

```diff
@@ -2,7 +2,7 @@ steps:
   - name: 'gcr.io/$PROJECT_ID/scala-sbt'
     args:
       - |
-        testOnly -- -l cronJobs.dbTest; janitor/docker:publishLocal
+        JVM_OPTS="-Xmx4G -Xms4G -Xss2M"; testOnly -- -l cronJobs.dbTest; janitor/docker:publishLocal
```

Comment:
@Qi77Qi would you happen to know how I could pass these arguments to the docker container?

Reply:
try sth like 
```
  env:
  - JAVA_OPTS: -Xmx4G -Xms4G -Xss2M
 ```

see [this](https://cloud.google.com/build/docs/configuring-builds/substitute-variable-values) for setting env in cloud build

Reply:
Ah thanks, looks slightly different from what I tried [here](https://github.com/broadinstitute/leonardo-cron-jobs/pull/289/commits/35e5e6db3c00d391280dfbad1a0f4bc3977f0eec), I will give it another try now :)

Reply:
That worked, thanks a lot!

Reply:
yea, it should be `JAVA_OPTS`